hourglass.py

```diff
@@ -123,6 +134,9 @@ def resume_train(self, batch_size, model_save_base_dir, model_json, model_weight
         print('Loss function selected: {}'.format(loss_str))
         self.loss = get_loss_from_string(loss_str)
```

Comment:
I may be missing something but I'm having a hard time finding where this loss value gets used.

Reply:
in `compile model`